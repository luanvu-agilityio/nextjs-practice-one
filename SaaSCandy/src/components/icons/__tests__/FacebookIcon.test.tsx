import { render } from '@testing-library/react';
import FacebookIcon from '../FacebookIcon';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<FacebookIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
