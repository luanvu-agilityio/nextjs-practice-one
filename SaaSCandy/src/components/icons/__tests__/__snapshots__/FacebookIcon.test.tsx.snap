// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon > matches snapshot 1`] = `
<svg
  fill="none"
  height="40"
  viewBox="0 0 40 40"
  width="40"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    height="39"
    rx="19.5"
    stroke="#EE7B11"
    width="39"
    x="0.5"
    y="0.5"
  />
  <path
    d="M22 21.5H24.5L25.5 17.5H22V15.5C22 14.47 22 13.5 24 13.5H25.5V10.14C25.174 10.097 23.943 10 22.643 10C19.928 10 18 11.657 18 14.7V17.5H15V21.5H18V30H22V21.5Z"
    fill="#EE7B11"
  />
</svg>
`;

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="40"
  viewBox="0 0 40 40"
  width="40"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    height="39"
    rx="19.5"
    stroke="#EE7B11"
    width="39"
    x="0.5"
    y="0.5"
  />
  <path
    d="M22 21.5H24.5L25.5 17.5H22V15.5C22 14.47 22 13.5 24 13.5H25.5V10.14C25.174 10.097 23.943 10 22.643 10C19.928 10 18 11.657 18 14.7V17.5H15V21.5H18V30H22V21.5Z"
    fill="#EE7B11"
  />
</svg>
`;
